dashboard/alerts/alert.py

```diff
@@ -86,30 +127,36 @@ def saveAlert(self):
         :return:
         """
         try:
-            # Open config file to get path
-            config_file = open('/etc/capstone-ids/config.json', 'r')
-            config_data = json.load(config_file)
-            path = config_data["alert_collection_path"]
-
-            # Default path
-            if path == "" or path is None:
-                path = "/etc/capstone-ids/alert_collection.json"
+            path = CONFIG.alert_collection_path
 
             # Open alert collection file to read
-            alert_collection = open(path, 'r')
-            alert_data = json.load(alert_collection)
+            alert_collection = None
+            try:
+                alert_collection = open(path, 'r')
+                try:
+                    alert_data = json.load(alert_collection)
+                except json.decoder.JSONDecodeError:
+                    alert_data = json.loads("{\n \"alerts\": []\n}\n")
+            except FileNotFoundError:
+                alert_data = json.loads("{\n \"alerts\": []\n}\n")
```

Comment:
Still do not agree with the need to error check here. The logic seems confusing to me since we're essentially checking for if the user has decided to break our program by deleting a core file for our logic...